lib/Settings/UserFlags.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace MailPoet\Settings;
+
+use MailPoet\Models\UserFlag;
+use MailPoet\WP\Functions as WPFunctions;
+
+class UserFlags {
+
+  /**
+   * @var array|null
+   */
+  private static $data = null;
```

Comment:
Why is this static? Have you read this comment? https://github.com/mailpoet/mailpoet/blob/master/lib/Settings/SettingsController.php#L155

It is static only temporarily in `SettingsController` but we can have this private here. This class is only instantiated in the DI container. And we shouldn't need `clear` method.